### PR TITLE
ui: fix transaction output for coinbase tx

### PR DESCRIPTION
**Pull request by chikeichan**
related to #217 

this fixes the issues where transaction is not showing the correct _value_ -- it does not fix the _categorization_ issue (i.e. **Airdrop** being displayed as **Mining Reward**)

### CODE REVIEW

File: app/ducks/walletActions.js [hunk 1]
```diff
@@ -332,7 +332,9 @@ async function parseInputsOutputs(net, tx) {
 
     // Track normal receive amounts for later
     if (covenant.action === 'NONE') {
-      totalValue += output.value;
+      if (await walletClient.hasAddress(output.address)) {
```
Comment (pinheadmz) [inline app/ducks/walletActions.js:335]: Isn't this the same as just checking if `output.path` is non-null?

Comment (chikeichan) [reply app/ducks/walletActions.js]: didn't know there is a `output.path` variable 😄  

fixed now! I also fix `wallet/service` to inflate transactions before returning.

**Review by sdtsui — APPROVED** [review]
Screen Shot 2020-11-09 at 12 19 57 AM

Getting this error when switching to mainnet from simnet, but I don't think it relates. Otherwise looks fine.  I suppose the plan is to do categorization another time?  I think folks will be happy with any improvement here.

**Comment by chikeichan** [comment]
I will checkout the error but pretty sure it's unrelated. And yep - i will fix categorization in another PR

**Pull request merged**